charts/nx-agents/values.yaml

```diff
@@ -11,6 +11,11 @@ naming:
   # Overrides the app name (defined in _helpers.tpl) used when installing the chart
   nameOverride: ''
 
+localExecutorBinaryStorage:
```

Comment:
this can just go under `controller`, no need for top level keys that related to specific features that will eventually become default anyway